### ISSUE TITLE
`day01` update summary and motivational scenario connecting the curricula

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/tutorials-early#33]
> The index or first episode can introduce a motivational scenario (like the one written in https://epiverse-trace.github.io/git-rstudio-basics/01-basics.html)

_Originally posted by @avallecam in https://github.com/epiverse-trace/tutorials/issues/66#issuecomment-1814575348_

Following a social-cognitive theory approach ([reference](https://journals.physiology.org/doi/full/10.1152/advan.00132.2015)), estimating the reproductive number (R) can be defined as a proximal goal that would enhance learner self-efficacy. It is a topic with less time to master with the most benefit in learners' work. Also, R is possibly the best-known estimate for outbreaks, it has strong anticipatory thoughts in this discipline. So, R could be a central topic point and a motivator to open gates for questions and data inputs needed in different scenarios to generate it, and different new outputs you can generate from it. Apart from different alternatives of how to calculate and visualize it with different methods.

In other words, through lessons, the learner will build up the first workflow with packages before and after, plus variations under different circumstances.

An order could be:
- Estimate Rt without package dependencies
- Read parameters from the literature
- Read a line list
- Clean raw data
- Validate a line list
- Create a transmissibility report
- Estimate severity
- Forecast cases
- Simulate scenarios
- Compare interventions
